nix/cells/cloud/pipelines.nix

```diff
@@ -54,6 +55,20 @@ let
       }));
 
   ciTasksSeq = taskSequence "ci/" ciTasks (__attrNames ciTasks);
+
+  benchmark = { config, ... }:
+    let
+      runner = cell.library.plutus-benchmark-runner {
+        PR_NUMBER = "4956";
```

Comment:
@Pacman99 here is where you pass me the params: Please let PR_NUMBER be a string